.travis.yml

```diff
@@ -1,29 +1,22 @@
 language: node_js
-cache:
-  directories:
-    - ~/.cache
-    - ~/.npm
 before_install: npm install -g npm@latest
-install: 
-  - npm ci
 jobs:
   include:
     - stage: test
+      install: npm ci
       before_script:
         - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
         - chmod +x ./cc-test-reporter
         - ./cc-test-reporter before-build
       script:
-       - npm run test
-       - xvfb-run npm run test:accessibility:ci
-      after_script: ./cc-test-reporter after-build -t lcov --debug --exit-code $TRAVIS_TEST_RESULT
+        - npm run test
+        - ./cc-test-reporter after-build -t lcov --debug --exit-code $TRAVIS_TEST_RESULT
+        - xvfb-run npm run test:e2e:storybook:ci
     - stage: test
-      script: xvfb-run npm run test:e2e:ci
-    - stage: test
-      script: xvfb-run npm run test:e2e:storybook:ci
-    - stage: test
-      install: npm install --production
-      script: npm run build
+      install: npm ci only=production
```

Comment:
The syntax is `npm ci --only=production` https://docs.npmjs.com/cli/install

Reply:
Done